shard.yml

```diff
@@ -14,9 +14,9 @@ targets:
     main: src/amber/cli.cr
 
 dependencies:
-  radix:
-    github: luislavena/radix
-    version: ~> 0.3.8
+  amber_router:
+    github: amberframework/router
+    commit: 5883a23
```

Comment:
@robacarp I think we should release a new amber_router version to avoid using commit

Reply:
@faustinoaq Yeah, when this gets settled, I'll make a real release tag and update here before merging.